_data/atm1504/atm1504.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Amartya Mondal
+institution: Indian Institute of Technology Patna
+profile_pic: atm1504.jpg
+quote: "Beauty begins the moment you decide to be yourself!"
```

Comment:
Please follow the given [instruction](https://github.com/education/graduation#3-add-your-profile-information) by removing the quotes to maintain the same format